frontend/src/components/Playback/PlayerElement.vue

```diff
@@ -15,8 +15,11 @@
         :loop="playbackManager.isRepeatingOnce"
         :class="{ 'uno-object-fill': playerElement.isStretched.value }"
         @loadeddata="onLoadedData">
+        <SubtitleTrack
+          v-if="subtitleSettings.state.enabled && playerElement.currentExternalSubtitleTrack?.parsed !== undefined" />
         <track
           v-for="sub in playbackManager.currentItemVttParsedSubtitleTracks"
+          v-else
           :key="`${playbackManager.currentSourceUrl}-${sub.srcIndex}`"
```

Comment:
@seanmcbroom This is our only blocker now then 🙄. I wonder if the issue is caused by some issue in the transferable (main <-> WebWorker "serialization" of objects). I added a `console.log` at the end of the `parseSsa` while playing the only movie I know of from my library that has SSA (it's late here so I will need to find more media tomorrow).

Can you help me debug if you're getting results correctly in the main thread? Perhaps something else was borked? The logic here is super straightforward, it's really strange that such a simple `if` broke this.

Reply:
Seems like its a problem between the weird behavior of the media element and the TextTrack api. I'd say to just load all text tracks onto the media element and set the mode to "disabled". This will stop the client from loading the cues and processing events (https://developer.mozilla.org/en-US/docs/Web/API/TextTrack/mode#disabled). 

As for the custom track element, I believe it just cannot be displayed inside of the video element sadly.

I'm going to push a commit that will hopefully address these issues and you can review it.

Reply:
I just realized the track element is set to disabled by default, so I've removed that portion of the code